app/src/lib/trampoline/trampoline-command-parser.ts

```diff
@@ -126,8 +126,17 @@ export class TrampolineCommandParser {
       )
     }
 
+    const trampolineToken = this.environmentVariables.get(
+      'DESKTOP_TRAMPOLINE_TOKEN'
+    )
+
+    if (trampolineToken === undefined) {
+      throw new Error(`The trampoline token is missing`)
+    }
+
     return {
       identifier,
+      trampolineToken,
```

Comment:
I included the trampoline token to the trampoline commands because it was very convenient.